packages/plugin-vue/src/index.ts

```diff
@@ -142,7 +150,9 @@ export default function vuePlugin(rawOptions: Options = {}): Plugin {
       }
     },
 
-    config(config) {
+    config(config, env) {
+      options.prebundle ??= env.command === 'serve'
```

Comment:
Why here enabled by default in serve mode only?

Reply:
The prebundle in build mode is unnecessary , so I set options.prebundle enabled in server mode, but since your next discussion is about disabled by default, it should be set `false` by default

Reply:
resolve in 8021394fcec29b0f79863ca0fc75bd4cf2c0c42d